docusearch/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Document(models.Model):
+    """ A model representing a document from an agency. """
+
+    text = models.TextField(
```

Comment:
Yeah, I thought of that too. This just seemed like the easiest way to get going. Short term, I don't think it matters too much. We're not doing anything with the text field that the database would cry about.